README.md

```diff
@@ -15,22 +15,26 @@ implementation, such as [up-transport-zenoh-cpp][zenoh-transport-repo].
 
 ### Requirements:
 - Compiler: GCC/G++ 11 or Clang 13
-- Conan : 1.59 or latest 2.X
+- pip : To install others
+- Conan(pip) : 1.59 or > 2.4.2
```

Comment:
Conan versions above 2.3.x are explicitly incompatible right now. Where is this Conan2 version number coming from?

I'm also not entirely sure conanfile.txt for this repo is still Conan1 compatible.

Reply:
Ya, I don't know.  This is what you mentioned in the Teams chat and what worked for me.  Also, I agree that we might remove the Conan 1.x comments.  However, I was chatting with someone on Yammer that couldn't get Conan 2.3.2 (His comment was: Normally we can only pull Python packages from Artifactory, and if it's not there, we have to get the open source team to scan and add it.).  I mentioned he could just get it from Github, but he said the CI's are going to need it so, he might as well get it figured out now anyway.  I'll go ahead and remove the 1.x stuff.

Reply:
Please don't change the requirements *at all* in the PR